### PR TITLE
GitHub sync

### LINKED ISSUE #1324: No access to new organization
**Describe the bug**
No access to the new organization i have been added to, even after i revoked the access tokens to the OAuth app, and relogged in github.

**Steps to reproduce the issue:**
Be added to a new org and check out the ui. The new org wont be available.

**Expected behavior**
I want to see the new org in my organization list on pipeline-ui after refreshing.

### LINKED ISSUE #1375: Synchronize Github organizations
Add new Github organizations

### PR DESCRIPTION
**Pull request by sagikazarmark**
Fixes #1375
Fixes #1324

### CODE REVIEW

File: api/organization.go [hunk 1]
```diff
@@ -159,16 +180,48 @@ func CreateOrganization(c *gin.Context) {
...
+	user := auth.GetCurrentUser(c.Request)
+	token, err := auth.GetUserGithubToken(user.ID)
+	if err != nil {
+		errorHandler.Handle(err)
+
+		c.JSON(http.StatusInternalServerError, common.ErrorResponse{
+			Code:    http.StatusInternalServerError,
+			Message: "failed to retrieve github token",
+			Error:   err.Error(),
+		})
```
Comment (bonifaido) [inline api/organization.go]: Missing return.

File: api/organization.go [hunk 2]
```diff
@@ -159,16 +180,48 @@ func CreateOrganization(c *gin.Context) {
...
+
+	err = a.githubImporter.ImportOrganizations(user, token)
+	if err != nil {
+		errorHandler.Handle(err)
+
+		c.JSON(http.StatusInternalServerError, common.ErrorResponse{
+			Code:    http.StatusInternalServerError,
+			Message: "syncronization failed",
+			Error:   err.Error(),
+		})
```
Comment (bonifaido) [inline api/organization.go]: Missing return.

File: api/organization.go (hunk 1 above)
Comment (sagikazarmark) [reply api/organization.go]: Thanks, fixed

File: api/organization.go (hunk 2 above)
Comment (sagikazarmark) [reply api/organization.go]: Thanks, fixed

**Pull request merged**